configJson.js

```diff
@@ -0,0 +1,24 @@
+const configBuilder = require('openfin-config-builder');
+
+if (process.argv.length <= 2) {
+    console.log('Usage: ' + __filename + ' <deployName>');
```

Comment:
`console.error` probably more appropriate here

Does `node` need to be used before the file name?

Can the string concatenation be replaced with template strings? (Below too.)